src/sonic-config-engine/tests/simple-sample-graph.xml

```diff
@@ -217,6 +217,7 @@
           <PortName>0</PortName>
           <Priority>0</Priority>
           <Speed>10000</Speed>
+          <Description>Interface description</Description>
```

Comment:
I thought about mixing interfaces with and without "description" to make sure that our code/testcases can deal with both scenarios.

Reply:
Sure. I'll remove two of them.

Reply:
Done